static/src/javascripts/projects/common/modules/identity/modules/modal.js

```diff
@@ -4,6 +4,35 @@ import fastdom from 'lib/fastdom-promise';
 const ERR_MODAL_NOT_FOUND = 'Modal not found';
```

Comment:
`modal.js` should be agnostic to the newsletters and only handle modals irregardless of content i'm afraid, so this whole thing needs a refactor :(